server/server.go

```diff
@@ -310,7 +312,10 @@ func NewServer(ctx context.Context, prof Profile) (*Server, error) {
 		return c.String(http.StatusOK, "OK!\n")
 	})
 	// Register pprof endpoints.
-	registerPProfEndpoints(e)
+	pprof.Register(e)
+	// Register prometheus metrics endpoint.
+	p := prometheus.NewPrometheus("echo", nil)
```

Comment:
qq: is echo a standard sub-system to use?

Reply:
It's form the example to distinguish it from other prometheus subsystems. we can see via：
> https://echo.labstack.com/middleware/prometheus/

![image](https://user-images.githubusercontent.com/16610542/174467337-9791a75d-5e07-4ea0-9063-16898b00ad44.png)

But it's really hard to answer strictly whether "echo" is a standard Prometheus subsystem. In addition to echo, we can also see http, gin, etc.

Reply:
How about using bb to indicate timeseries from Bytebase?

Reply:
Maybe you can, but I don't think that's appropriate.

We can see the default subsystem name is "echo"：
https://github.com/labstack/echo-contrib/blob/master/prometheus/prometheus.go#L40

But "defaultSubsystem" is only used as the subsystem name, and the given one is used to register the metrics：
https://github.com/labstack/echo-contrib/blob/master/prometheus/prometheus.go#L170-L200

Maybe I will use different prometheus subsystem names to distinguish ”/api/v1“ and ”/api/v2“, but the use of bb here doesn't seem to have any special purpose. it just tells us that this is bytebase.

Reply:
I see. How about "api" if that's better?